server/src/test/java/org/elasticsearch/index/engine/LiveVersionMapTestUtils.java

```diff
@@ -93,6 +95,14 @@ public static boolean isSafeAccessRequired(LiveVersionMap map) {
         return map.isSafeAccessRequired();
     }
 
+    public static void enforceSafeAccess(LiveVersionMap map) {
+        map.enforceSafeAccess();
+    }
+
+    public static Map<BytesRef, VersionValue> getMap(LiveVersionMap.VersionLookup versionLookup) {
+        return versionLookup.getMap();
+    }
+
```

Comment:
Is this unused or do you need them in stateless?

Reply:
Unused. Removed it.